server/server.js

```diff
@@ -15,6 +15,7 @@ const methodOverride = require('method-override');
 const app = module.exports = loopback();
 const boot = require('loopback-boot');
 const cookieParser = require('cookie-parser');
+const debug = require('debug')('workspace:server');
```

Comment:
So far, our convention has been to prefix all loopback-related debug logs with `loopback:` prefix, see e.g. https://github.com/strongloop/loopback/blob/ab2e462ca673cca5deb3c7e3ed2cd3829960d030/lib/model.js#L12

```suggestion
const debug = require('debug')('loopback:workspace:server');
```